packages/filecoin-actor-utils/test/index.spec.ts

```diff
@@ -34,7 +33,7 @@ describe('utils', () => {
 
     test('should find the mainnet actor code when not providing a network', () => {
```

Comment:
Might be better to change the name of this tests, as now you are finding a code

Reply:
Also good call